dvc/repo/__init__.py

```diff
@@ -257,17 +257,19 @@ def used_cache(
         cache["azure"] = []
 
         for branch in self.scm.brancher(
-            all_branches=all_branches, all_tags=all_tags
+            all_branches=all_branches, all_tags=all_tags, checkout=False
         ):
             if target:
                 if recursive:
-                    stages = self.stages(target)
+                    stages = self.stages(target, branch=branch)
```

Comment:
does `branch` mean actual branch or more like a `revision` in Git terminology?

Reply:
latter, but I think it is ok to use the `branch` for naming here... but `rev` would be shorter... hm...